src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.ImportPublishProfile.targets

```diff
@@ -0,0 +1,31 @@
+<!--
+***********************************************************************************************
+Microsoft.NET.Sdk.ImportPublishProfile.targets
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (c) .NET Foundation. All rights reserved.
+***********************************************************************************************
+-->
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <!-- Default to having imported the publish profile so the Web SDK doesn't also attempt to do so. -->
+    <PublishProfileImported>true</PublishProfileImported>
```

Comment:
There is a lot of default property. If any one of these property is wrong, it will fail the build? If so the test is good enough

Reply:
Altering most of these properties would result in the wrong path to the import profile the tests that were added create, so it will cause them to fail because the wrong output from the publish will occur (primarily the publish directory won't exist since it'll be RID-less).